proposals/lip-0047.md

```diff
@@ -517,17 +515,16 @@ Let `genesisBlockAssetBytes` be the bytes included in the `data` property of a d
   * Check that the `blsKey` property of every entry in the `assetPoA.validators` array has length 48.
   * Check that the `proofOfPossession` property of every entry in the `assetPoA.validators` array has length 96.
 
-* Checks for the `assetPoA.snapshotSubstore`: 
+* Checks for the `assetPoA.snapshotSubstore` (note that these checks are equivalent to those for the verification of the Authority Update command): 
```

Comment:
I know that you added the statement about the equivalence based on a comment from Jan. But I think it is misleading as it is not to 100% equivalent as you pointed out. Maybe "similar", "almost identical" or "almost equivalent". Or you could just drop it.

Reply:
changed to "similar"